ch_pipeline/analysis/flagging.py

```diff
@@ -189,11 +189,14 @@ def _combine_st_mad_hook(self, times):
 
         return madtimes
 
-    def _static_rfi_mask_hook(self, freq):
+    def _static_rfi_mask_hook(self, freq, timestamp=None):
         """Use the static CHIME RFI mask.
 
         Parameters
         ----------
+        timestamp : float
+            Start observing time (in unix time)
+
```

Comment:
This parameter should be below "freq" so it matches the order in which they're provided to the function.